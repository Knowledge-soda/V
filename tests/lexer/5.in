ifec(<(a, 3), {x := +(x, 5)}, {x := *(x,)}, a)
