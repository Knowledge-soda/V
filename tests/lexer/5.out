NAME(ifec) BEGIN NAME(<) BEGIN NAME(a) SEP NUM(3) END SEP SUB_BEG NAME(x) DEF NAME(+) BEGIN NAME(x) SEP NUM(5) END SUB_END SEP SUB_BEG NAME(x) DEF NAME(*) BEGIN NAME(x) SEP END SUB_END SEP NAME(a) END EOL
